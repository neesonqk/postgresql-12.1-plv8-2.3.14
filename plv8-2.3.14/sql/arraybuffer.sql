CREATE FUNCTION crash_test() RETURNS boolean
LANGUAGE plv8 IMMUTABLE STRICT
AS $$
  var a = new ArrayBuffer(8);
  return true;
$$;

SELECT crash_test();
