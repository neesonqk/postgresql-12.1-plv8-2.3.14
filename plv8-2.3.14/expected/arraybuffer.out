CREATE FUNCTION crash_test() RETURNS boolean
LANGUAGE plv8 IMMUTABLE STRICT
AS $$
  var a = new ArrayBuffer(8);
  return true;
$$;
SELECT crash_test();
 crash_test 
------------
 t
(1 row)

